easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,39 @@
+name = 'netCDF'
+version = '4.4.1.1'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.unidata.ucar.edu/downloads/netcdf/ftp/']
+
+dependencies = [
+    ('cray-hdf5/1.10.0', EXTERNAL_MODULE),
```

Comment:
`cray-hdf5/1.10.0` conflicts with `cray-hdf5/1.10.0.3`
in CDO-1.9.0-CrayGNU-17.08.eb ?

Reply:
Done.